tests/wpt/web-platform-tests/css/cssom/stylesheet-owner.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<head>
+  <meta charset="utf-8">
+  <title>insertrule @import test</title>
+  <link rel="match" href="stylesheet-owner-ref.html">
+  <style><link rel="stylesheet" ></style>
```

Comment:
Maybe no need to use the `style` and `link` here 👀? (The broken syntax highlight makes me notice it)

Reply:
So, an empty style tag would be fine?

Reply:
After reading https://github.com/servo/servo/pull/23073#issuecomment-475346694 again, I think yes.

As @jdm mentioned in step 1, there's a `style` element and then we use `insertRule` API to add new rules into it.

Besides, I think having `<link rel="stylesheet" >` would be invalid syntax 😂?

Reply:
ok thanks